src/cargo/ops/registry.rs

```diff
@@ -334,12 +335,13 @@ pub fn registry_configuration(
     Ok(RegistryConfig { index, token })
 }
 
-pub fn registry(
+fn registry(
```

Comment:
`fn registry` isn't called outside this file and is already a bit gnarly and specific in its usage, so I thought it might be best made private.